src/Ledger/Certs.lagda

```diff
@@ -161,6 +173,16 @@ stake distribution anymore. Genesis delegations and MIR certificates
 have been superceded by the new governance mechanisms, in particular
 the \TreasuryWdrl governance action in case of the MIR certificates.
 
+\subsection{Delegation}
+
+Registered credentials can now delegate to a DRep as well as to a
+stake pool. This is achieved by giving the \delegate certificate two
+optional fields, corresponding to a DRep and stake pool. Stake can be
+delegated for voting and block production simultaneously, since these
+are two separate features. In fact, preventing this could weaken the
+security of the chain, since security relies on high participation of
+honest stake holders.
+
```

Comment:
Would be good to mention that the staked ADA counts towards voting power for both dRep and stake pool at the same time. (I understand it is what is implied already)